admin/client/src/state/schema/SchemaReducer.js

```diff
@@ -10,6 +11,16 @@ export default function schemaReducer(state = initialState, action = null) {
       return deepFreeze(Object.assign({}, state, { [action.payload.id]: action.payload }));
     }
 
+    case ACTION_TYPES.CLEAR_MESSAGE: {
```

Comment:
A whole form `render()` process happens with either a `setState()` call or a `schema` state change, it's how diffing for any DOM changes work.

I had considered using `setState()` in the `Form` class, which to me made more sense than `FormBuilderLoader`. Using `CLEAR_MESSAGE` in redux state gives a more predictable outcome than setting state, and since the message wasn't showing again until after the form was submitted, in which case you receive a new message and `schema` state to display.